src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseFVM.cpp

```diff
@@ -305,12 +305,12 @@ void SinglePhaseFVM< BASE >::applySystemSolution( DofManager const & dofManager,
   } );
 }
 
-template< >
-void SinglePhaseFVM< SinglePhaseBase >::assembleFluxTerms( real64 const dt,
```

Comment:
I am trying to understand the logic here. what is proposed is always using BASE besides singlePhaseProppantBase specialization? And I guess this implementation is assumed not only being restricted to SinglePhaseBase.

Reply:
If I remember correctly, the change should be just related to the default template argument (`SinglePhaseBase`), no changes in other logic. `SinglePhaseFVM< SinglePhaseProppantBase >::assembleFluxTerms` is still down below.